lib/Client.js

```diff
@@ -111,6 +111,7 @@ export class Client {
       invoiceId: httpResponse.headers.szlahu_szamlaszam,
       netTotal: httpResponse.headers.szlahu_nettovegosszeg,
       grossTotal: httpResponse.headers.szlahu_bruttovegosszeg,
+      customerAccount: httpResponse.headers.szlahu_vevoifiokurl,
```

Comment:
```suggestion
      customerAccountUrl: httpResponse.headers.szlahu_vevoifiokurl,
```

I think the `Url` suffix more future proof.